### PR TITLE
VACMS-4022: Fix accessibility test failure

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #4022

## Testing done

Automated testing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_backend/va_gov_backend.libraries.yml [hunk 1]
```diff
@@ -31,6 +31,10 @@ gtm_tag_trackers:
   version: 1.x
   js:
     js/gtm_tag_trackers.js: {}
+  dependencies:
+    - core/drupal
+    - core/jquery
+    - core/jquery.once
```
Comment (oksana-c) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.libraries.yml:37]: do we need these dependencies?

Comment (acrollet) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.libraries.yml:37]: yep, a js error occurs on the login page without the .once dependency.

**Review by ndouglas — APPROVED** [review]
Thanks again for the block visibility additions.

**Review by ElijahLynn — APPROVED** [review]

File: scripts/tests/cypress-axe.sh [hunk 2]
```diff
@@ -12,5 +12,8 @@ if [ -n "$CMS_IS_BRD" ]; then
   exit 0
 fi
 
+# Prime the login page cache
+curl http://localhost &> /dev/null
+
```
Comment (ElijahLynn) [inline scripts/tests/cypress-axe.sh:17]:
```suggestion
```

This was for debugging the first run failure, says Adrian.

**Review by ElijahLynn — APPROVED** [review]

File: .tugboat/config.yml [hunk 3]
```diff
@@ -124,10 +124,11 @@ services:
       online:
         # We only want tests running on PRs, not branches like the base preview build of master.
         # This sets the commit status checks to pending all at once.
-        - if [ "$TUGBOAT_PREVIEW_TYPE" = "pullrequest" ]; then composer yaml-test va/web/build va/tests; fi
+        # FIXME: verbose flag for debugging
```
Comment (acrollet) [inline .tugboat/config.yml:127]:
```suggestion
```

remove before merging

**Comment by acrollet** [comment]
Verified passing tests:

Screen Shot 2021-01-15 at 3 14 36 PM

**Pull request merged**